lib/pavilion/dir_db.py

```diff
@@ -92,24 +97,73 @@ def create_id_dir(id_dir: Path) -> (int, Path):
 
         return next_id, next_id_path
 
-
 def default_filter(_: Path) -> bool:
     """Pass every path."""
 
     return True
 
+def is_hex(string: str) -> bool:
+    """Determines if passed string is valide hex or not.
+    :param string: A string to check.
+    :return bool: True if hex, False, if not hex.
+    """
+
+    try:
+        int(string, 16)
+    except ValueError as err:
+        return False
+
+    return True
 
 def select(id_dir: Path,
-           filter_func: Callable[[Path], bool] = default_filter) -> List[Path]:
+           filter_func: Callable[[Path], bool] = default_filter,
+           builds_dir: bool = False) -> List[Path]:
```

Comment:
Don't tie this to the idea that it's a build directory, make it a generic flag to indicate hex filenames.